Export Access API from index (#861)
inrupt/solid-client-js: CHANGELOG.md

## [Unreleased]

### New features

> - `access_api`: A set of APIs to manage Access which are agnostic to the Access

Reviewer: Sorry, the ticket was unclear here, but since this object is a workaround for people who aren't able to use exports maps, as part of this PR we should also make the universal access module available through the exports map (i.e. using the "New feature" PR checklist).

And since that's the recommended way to import it (if your tooling supports it, which most tooling will as time goes by), we should just mention that module here and only mention this object in the docs with a note on compatibility.

And maybe explicitly mention that the Access Control mechanism currently supported are the only two mechanisms for Solid currently known, i.e. Web Access Control and Access Control Policies. Or would that be too detailed and something better suited for a note somewhere in the docs?

Author: Fixed respectively in de729cf17da372ad5b87e27a1b4be238ab95e6c5 and 35a8b79d33792e4e0b2565da636a76cff5aa0b29. Thanks for the detailed explanations !

## [Unreleased]

### New features

> - `@inrupt/solid-client/access/universal` and `@inrupt/solid-client/access/for`:

Reviewer: ```suggestion
- `@inrupt/solid-client/access/universal`:
```

The former re-exports the latter, so there's no need to expose the latter :)

Author: If you only used `getAccessFor`, and directly imported `@inrupt/solid-client/access/for`, wouldn't you be able to tree-shake `@inrupt/solid-client/access/universal` ? It's not like it was a massive gain, but that was my reasoning initially ^^

Reviewer: Most bundlers are luckily smarter than that and can statically analyse code on the function level, i.e. if you only import a single function from a module, and that doesn't reference other functions in that module and doesn't have side effects, it will drop all other code from that module. (This only works with ES modules though.)

> - `@inrupt/solid-client/access/universal` and `@inrupt/solid-client/access/for`:
>   A set of APIs to manage Access which are agnostic to the Access Control mechanism

Reviewer: Also, maybe we should mention here too that this is a preview release?

```suggestion
- `@inrupt/solid-client/access/universal`: an early preview of
  a set of APIs to manage Access which are agnostic to the Access Control mechanism
```

Possibly even noting the fact that there are some bugs that are yet to be fixed, but that they only occur in a limited number of edge cases involving existing Access Policies.

  actually implemented (from the user's point of view). Currently, these modules

>   support the two know access controls for Solid, i.e. Web Access Control and

Reviewer: ```suggestion
  support the two known access controls for Solid, i.e. Web Access Control and
```
